### ISSUE TITLE
Tests failures

### ISSUE THREAD

**Issue by stuarteberg** [svenpeter42/fastfilters#7]
When I build the conda recipe, I see a test failure in a test that compares with vigra.  Since the `fastfilters` code hasn't changed in a while, the failure is presumably caused by some change in vigra.  Do you have any clue what could be going wrong?

```
TEST START: fastfilters-0.2.4-py27_0_g649a633
updating index in: /miniconda2/conda-bld/osx-64
updating index in: /miniconda2/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    fastfilters-0.2.4          |  py27_0_g649a633         531 KB  local

The following NEW packages will be INSTALLED:

    boost:       1.55.0-5                            ilastik
    fastfilters: 0.2.4-py27_0_g649a633               local
    fftw:        3.3.4-1                             ilastik
    h5py:        2.6.0-np111py27_1
    hdf5:        1.8.16-0
    jbig:        2.1-0                               ilastik
    jpeg:        8d-2
    lemon:       1.2.4-2                             ilastik
    libgcc:      4.8.5-1
    libpng:      1.6.17-0
    libtiff:     4.0.6-1                             ilastik
    mkl:         11.3.3-0
    nose:        1.3.7-py27_1
    numpy:       1.11.2-py27_0
    openssl:     1.0.2j-0
    pip:         9.0.1-py27_1
    python:      2.7.13-0
    readline:    6.2-2
    setuptools:  27.2.0-py27_0
    six:         1.10.0-py27_0
    sqlite:      3.13.0-0
    tk:          8.5.18-0
    vigra:       1.11.0.post199-np111py27_0_g48b7d43 local
    wheel:       0.29.0-py27_0
    xz:          5.2.2-1
    zlib:        1.2.8-3


WARNING: conda-build appears to be out of date. You have version 2.0.10 but the
latest version is 2.1.0. Run

conda update -n root conda-build

to get the latest version.

+ source /miniconda2/bin/activate /miniconda2/conda-bld/fastfilters_1483747753253/_t_env
+ /miniconda2/conda-bld/fastfilters_1483747753253/_t_env/bin/python -s /miniconda2/conda-bld/fastfilters_1483747753253/test_tmp/run_test.py
===== testing package: fastfilters-0.2.4-py27_0_g649a633 =====
import: u'fastfilters'
===== fastfilters-0.2.4-py27_0_g649a633 OK =====
+ /bin/bash -x -e /miniconda2/conda-bld/fastfilters_1483747753253/test_tmp/run_test.sh
+ python /miniconda2/conda-bld/fastfilters_1483747753253/work/tests/vigra_compare.py compare-success.txt
('gaussian ', 0, 1.0, 2.3841858e-07)
('gaussian ', 0, 5.0, 1.1920929e-07)
('gaussian ', 0, 10.0, 5.2154064e-08)
('gaussian ', 1, 1.0, 1.1920929e-07)
('gaussian ', 1, 5.0, 0.00011882256)
Traceback (most recent call last):
  File "/miniconda2/conda-bld/fastfilters_1483747753253/work/tests/vigra_compare.py", line 26, in <module>
    raise Exception("FAIL: ", order, sigma, np.max(np.abs(res_ff - res_vigra)))
Exception: ('FAIL: ', 1, 5.0, 0.00011882256)
TESTS FAILED: fastfilters-0.2.4-py27_0_g649a633
```

**Comment by svenpeter42** [comment]
Sorry, I must've missed this issue - is this still happening as of today?

**Comment by svenpeter42** [comment]
fixed with 90bd57a
